### DISCUSSION TITLE
Naming plates

### DISCUSSION THREAD

**Discussion by IrishMoss** [OrcaSlicer/OrcaSlicer#570] [Ideas]
Not sure how much interest these is for this one, but I would find it really helpful if there was a way to rename plates.
Might sound trivial, but it would be helpful for large projects where you have many plates of various objects for a large project.

**Comment by luchfilip** [comment]
this was actually done. https://github.com/SoftFever/OrcaSlicer/issues/1010 

Bottom icon
<img width="112" alt="image" src="https://user-images.githubusercontent.com/5247619/236985461-aa50f399-66cd-44af-9f80-5f55059b27f6.png">

**Comment by jhonnyjoao** [comment]
Not only that! I think it would be interesting to be able to change the printer for each table! It would be very useful for those who have more than one printer

**Comment by julie777** [comment]
It is there, but not where you would expect. I already created an issue with much more detail about naming plates.  #1010
